src/modules/70-pipeline/components/PipelineSteps/Steps/CustomVariables/CustomVariablesEditableStage.tsx

```diff
@@ -94,7 +94,7 @@ export function CustomVariablesEditableStage(props: CustomVariableEditableProps)
       validate={debouncedUpdate}
       validationSchema={enableValidation ? getValidationSchema(getString) : undefined}
     >
-      {formik => {
+      {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
      {(formik: unknown) => {
```